Flutter/cartesian-charts/zoom-pan.md

```diff
@@ -48,6 +48,52 @@ Pinch zooming can be enabled by [`enablePinching`](https://pub.dev/documentation
 {% endhighlight %}
 {% endtabs %}
 
+## Directional zooming
+
+The directional zooming feature improves the zoom experience by allowing users to zoom in and out in a specific direction (horizontal, vertical, or diagonal) depending on the angle of their pinch or drag gesture. To enable this feature, set the [`enableDirectionalZooming`]() property to `true`, the default value is `false`. The following code sample demonstrates how to enable directional zooming.
+
+{% tabs %}
+{% highlight dart hl_lines="7" %}
+
+    late ZoomPanBehavior _zoomPanBehavior;
+    
+    @override
+    void initState(){
+      _zoomPanBehavior = ZoomPanBehavior(
+                  // Enables directional zooming
+                  enableDirectionalZooming: true,
+                  enablePinching: true,
+                  zoomMode: ZoomMode.xy,
+                );
+      super.initState();
+    }
+
+    @override
+    Widget build(BuildContext context) {
+      return Scaffold(
+        body: SafeArea(
+          child: Center(
+            child: Container(
+              height: 300, 
+              width: 350, 
+              child: SfCartesianChart(
+                zoomPanBehavior: _zoomPanBehavior
+              )
+            )
+          )
+        )
+      );
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+>**Note**: 
```

Comment:
and [`zoomMode`]() is set to [`ZoomMode.xy`]().

is this need to tell, any how if customer set x mean he only expect the x direction alone. so no need this.